web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/svp/SvpAvklartOppholdPeriodeDto.java

```diff
@@ -6,5 +6,7 @@
 
 import no.nav.foreldrepenger.behandlingslager.behandling.tilrettelegging.SvpOppholdÅrsak;
 
-public record SvpAvklartOppholdPeriodeDto(@NotNull LocalDate fom, @NotNull LocalDate tom, @NotNull SvpOppholdÅrsak oppholdÅrsak, boolean forVisning) {
+public record SvpAvklartOppholdPeriodeDto(@NotNull LocalDate fom, @NotNull LocalDate tom, @NotNull SvpOppholdÅrsak oppholdÅrsak, SvpOppholdKilde oppholdKilde, boolean forVisning) {
```

Comment:
Burde svpOppholdKilde også være not null?

Reply:
Burde vi ha disse annotasjonene her i det hele tatt - jeg tror ikke det er satt opp validering av respons-dtoer, kun inngående? Så denne valideringen slår vel uansett ikke til.

Reply:
Kanskje bedre å validere i konstruktøren om feltene skal være satt.

Reply:
Ser den kommer inn for å bekrefte også. Mener jeg satte den slik i en expand-contract type ting ettersom frontend må komme etter.

Reply:
Ser frontend sender den selv om det er et for frontend ukjent felt. Jeg legger på NotNull, og mapper enn så lenge manuelt til REGISTRERT_AV_SAKSBEHANDLER og gjør en custom sjekk av likhet for å finne hva som er endret.